GOVERNANCE.md

```diff
@@ -0,0 +1,75 @@
+
+Our goal is to encourage a diverse community of contributors and users to participate in the development of the Vega projects.
+
+## People
+
+We define three levels of people in the Vega projects: contributors, maintainers, and admins. We expect everyone to adhere to our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Contributors
+
+Contributors are anyone who makes a contribution to a Vega project via GitHub. This includes filing issues, sending pull requests, and participating in discussions. There are no requirements to participate and we encourage everyone to contribute. You do not have to be a maintainer to send a pull request.
+
+### Maintainers
+
+Maintainers belong to all or some subset of Vega projects. Maintainers have write access and can push directly to branches on GitHub. They are expected to follow development guidelines, review pull requests, and participate in discussions. Maintainers also have access to specific developer channels on Slack.
+
+To become a maintainer, a contributor must be nominated by an existing maintainer and approved by a majority of maintainers via Slack reactions within 7 days (simple majority of responses). A new maintainer should be added to 1) the GitHub team, 2) the Slack channel, and 3) the appropriate list below. We want more people to become maintainers and lower the barrier the enter. If you are interested in becoming a maintainer, please reach out to an existing maintainer.
+
+Maintainers who are inactive for more than six months may be removed from the maintainer list.
+
+[Admins](#admins) are also maintainers.
+
+#### [Vega](https://github.com/vega) GitHub Organization Maintainers
+
+* @fandu-db
+* @joelostblom
+* @jonmmease
+* @keckelt
+* @mattijn
+* @yhoonkim
+
+#### [Altair-Viz](https://github.com/altair-viz) GitHub Organization Maintainers
+
+* @jonmmease
+
+### Admins
+
+Admins are maintainers who can also make releases. Admins are [owners of the GitHub organization](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#organization-owners).
+
+To become an admin, a maintainer must be nominated and approved by the existing admins using the same process as for maintainers.
+
+Admins who are inactive for a year may be removed from the admin list.
+
+The current admins are:
+
+#### [Vega](https://github.com/vega) GitHub Organization Admins
+
+* @arvind
+* @domoritz
+* @jheer
+* @kanitw
+* @lsh
+
+#### [Altair-Viz](https://github.com/altair-viz) GitHub Organization Admins
+
+* @binste
+* @ChristopherDavisUCI
+* @jakevdp
+* @joelostblom
+* @mattijn
+* @stsievert
```

Comment:
@binste @ChristopherDavisUCI @jakevdp @joelostblom @mattijn @stsievert please approve that you want to be an admin

Reply:
Resolved since Altair has their own docs for now.